examples/bench_conv2d_sigmoid.py

```diff
@@ -65,82 +65,82 @@
 
     def provide_input_conv11weights() -> tf.Tensor:
         w = tf.random_normal(shape=conv11_fshape, dtype=tf.float32)
-        return tf.Print(w, [w], message="w11:")
+        return tf.print(w, [w], message="w11:")
```

Comment:
These will need to be changed a bit because `tf.print` doesn't return anything.

`tf.Print` returned a tensor that would put a printing op into the graph but otherwise the tensor worked the same. Since `tf.print` returns nothing we will still want to return `w` above

```python
w = tf.random_normal(shape=conv11_fshape, dtype=tf.float32)
tf.print(w, [w], message="w11:")
return w
```

Likewise for everything below

Reply:
alright , I will make the changes.

Reply:
I will also think about whether we need to continue support for 1.9 and comment on the issue.

Reply:
> These will need to be changed a bit because `tf.print` doesn't return anything.
> 
> `tf.Print` returned a tensor that would put a printing op into the graph but otherwise the tensor worked the same. Since `tf.print` returns nothing we will still want to return `w` above
> 
> ```python
> w = tf.random_normal(shape=conv11_fshape, dtype=tf.float32)
> tf.print(w, [w], message="w11:")
> return w
> ```
> 
> Likewise for everything below

@bendecoste Sorry for the delay , was busy finishing the semester. I have made a few changes , let me know if those changes were done right. If , yes I will continue to cover rest of the codebase.

Reply:
Do I have to modify the .rst files as well?